x/distribution/client/cli/tx.go

```diff
@@ -23,6 +23,11 @@ var (
 	flagOnlyFromValidator = "only-from-validator"
 	flagIsValidator       = "is-validator"
 	flagComission         = "commission"
+	flagMaxMessagesPerTx  = "max-msgs"
+)
+
+const (
+	MaxMessagesPerTxDefault = 5
```

Comment:
Will this break multimsg `withdraw-all-rewards` txs?

Reply:
What do you mean? If you have more than 5 delegations, it'll just create and broadcast multiple txs afaik. Correct me if I'm wrong @jleni.

Reply:
👌

Reply:
yes, it will withdraw all but it broadcast several txs.